apps/paymaster-proxy/src/jsonRpc/createExpressRouterFromJsonRpcRouter.ts

```diff
@@ -0,0 +1,41 @@
+import type { NextFunction, Request, Response } from 'express'
+import express from 'express'
+
+import { JsonRpcError } from '@/errors/JsonRpcError'
+import type { JsonRpcRouter } from '@/jsonRpc/JsonRpcRouter'
+
+export const createExpressRouterFromJsonRpcRouter = (
+  jsonRpcRouter: JsonRpcRouter,
+) => {
+  const router = express.Router()
+
+  router.use('/', (req, res, next) => {
+    // reject non-JSON requests
+    if (req.headers['content-type'] !== 'application/json') {
+      return res.status(400).send()
+    }
+    next()
+  })
+
+  router.use('/', express.json())
```

Comment:
Json RPC Parse error is handled here instead of at JsonRpcRouter since it is closely coupled to the bodyparsing logic in express. kept it here to prevent JsonRpcRouter from being tightly coupled with express.

another option is to just pass the raw buffer to the JsonRpcRouter and let it handle the parsing logic, but adds complexity so decided against in favor of just using express.json()